pkg/api/v20170701/types.go

```diff
@@ -115,8 +115,7 @@ const (
 // OrchestratorProfile contains Orchestrator properties
 type OrchestratorProfile struct {
```

Comment:
This needs to be changed for ManagedCluster type as well (2017-08-31 version).

Reply:
Ah, i missed it. Just found it is called `KubernetesRelease`. Will make the change.